client/src/components/campaignDisplay/campaignTabs.tsx

```diff
@@ -4,12 +4,19 @@ import useAxios from 'axios-hooks';
 import { settlementApi } from '../../service/api';
 import TimelineTab from './TimelineTab';
 import StorageTab from './StorageTab';
+import { useEffect } from 'react';
 
 const CampaignTabs = () => {
     const { id } = useParams();
     const [{ data, loading, error }, refetch] = useAxios(`${settlementApi}/get/${id}`);
     const navigate = useNavigate();
 
+    useEffect(() => {
+        if (data) {
+            console.log('Settlement Data', data.settlement);
+        }
+    }, [data]);
+
```

Comment:
Don't forget to get rid of this once you're done with it.

Reply:
using this for ongoing testing, so i can better read the retrieved data. will be removed at a later time.